inst/mlr3shiny/server/server_predict.R

```diff
@@ -421,4 +420,13 @@ get_final_training_code <- function(task, learner) {
   final_train_code <- paste0(final_train_code, "learner$predict(task) <br>")
   }
   return(final_train_code)
+}
+
+reset_trained_learner_list <- function () {
+  # iterating over all trained learners and resetting them
+  # See: https://stackoverflow.com/questions/61887112/how-to-reset-reactivevalues
+  for (trained_learner in names(trained_learner_list)) {
+    trained_learner_list[[trained_learner]] <- NULL
+  }
+  print(trained_learner_list)
```

Comment:
What is this print statement for?

Reply:
It was used for debugging during development 😆 it can be removed